Education/CodingStandards.md

```diff
@@ -183,26 +195,32 @@ This will alow you to reference said template anywhere within the page.  Any add
 ### 4.3. 
 
 ## 5. Custom Actions
-Because Custom Actions can contain code/scripts it creates a large amount of locations that a potentially buggy script could be hiding. For this reason, if your code will contain more than 5 lines, it is best to house your code in a global script, and pass it a reference to the quote, as necessary.
+Because Custom Actions can contain code/scripts it creates a large amount of locations that a potentially buggy script could be hiding. For this reason, if your code will contain more than a single line, it is best to house your code in a global script, and pass it a reference to the quote, as necessary.
 
 ```Python
 ScriptExecutor.ExecuteGlobal('GlobalScriptName', context.Quote)
 ```
 
-## 6. Tags
+## 6. Products
+Because Products can contain code/scripts it creates a large amount of locations that a potentially buggy script could be hiding. For this reason, if your code will contain more than a single line, it is best to house your code in a global script, and pass it a reference to the Product, as necessary.
+
+```Python
+ScriptExecutor.ExecuteGlobal('GlobalScriptName', Product)
+```
+## 7. Tags
 
 Tags exist as a faster way to access and process data related to quotes and products. They are more limited in terms of functionality but are much more performant. If you have the option to use a tag instead of a script, use a tag.
```

Comment:
maybe give some common example tags we can use

Reply:
this is not an educational doc, but instead instructions on standards.  But I did show/add some examples of nesting ctx tags inside C and Q tags